application_test.go

```diff
@@ -120,6 +120,10 @@ func addMinimalApplication(model Model) {
 	u.SetAgentStatus(minimalStatusArgs())
 	u.SetWorkloadStatus(minimalStatusArgs())
 	u.SetTools(minimalAgentToolsArgs())
+
+	if model.Type() == CAAS {
```

Comment:
This doesn't go here - it's an IAAS application we are adding ^^^^^^
This block shouldn't be needed